README.md

```diff
@@ -1,90 +1,31 @@
-trips|A list of trips that this User has taken as a passenger
+## At a Glance
 
-User Methods
+- Pair, [stage 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md#stage-2) project
+- Due before class, *DATE HERE*
 
-**Method**|**Description**
------|-----
-add_trip|Adds a trip to the user's list of trips
+## Introduction
 
+Remember the ride share exercise we did with designing and creating a system to track the ride share data from a CSV file? We did a lot of great work on creating arrays and hashes of data, but we've learned a lot since then!
 
-#### Trip
-A `Trip` represents a journey a `User` has taken with the service.
-
-Each `Trip` has:
-
-**Attribute**|**Description**
------|-----
-id|The Trip's ID number
-passenger|The User being transported on the trip
-start_time|When did this trip begin?
-end_time|When did this trip finish?
-rating|The rating given by the User, a number 1-5
-cost|How much did the passenger pay?
-
-#### TripDispatcher
-The `TripDispatcher` class is designed to load trips & users from a CSV file and provides methods to find and add trips.
-
-The `TripDispatcher` has:
-
-**Attribute**|**Description**|**Returns**
------|-----|-----
-passengers|A list of all users in the system|a collection of User instances
-trips|A list of all trips taken in the system|a collection of Trip instances
+Now, we're going to use our understanding of classes, methods and attributes to create an object-oriented implementation of our ride share system.
 
-The `TripDispatcher` has the following responsibilities:
--   load collections of `User`s, and `Trip`s from CSV files
--   store and manage this data into separate collections
+## Learning Goals
 
-The `TripDispatcher` does the following:
--   on instantiation, loads and creates `Trip`s, and `User`s, and stores them into collections
+- Reinforce and practice Ruby programming fundamentals, particularly
+  - Creating and instantiating classes with attributes
+  - Using composition to connect different classes together
+  - Working with files and large amounts of data
+  - Writing pseudocode and creating tests to drive the creation of our code
+- Quickly become familiar with a large and complex code base
+- Use inheritance to extend existing classes
 
-The `TripDispatcher` instance is able to:
+## Objective
 
-**Methods**|**Description**
------|-----
-load_users|A method which loads users from a CSV file, returning the list
-load_trips|A method which loads trips from a CSV file, returning the list
-find_passenger  |  find an instance of `User` given an ID
+We will build a Ruby library that loads lists of passengers, trips and drivers from CSV files, provides methods for exploring this data, and provides a way to request a new trip.
 
-By the end of this project, a `TripDispatcher` will be able to:
--   create new trips assigning appropriate passengers and drivers
+We will **not** write an interactive command-line program.
```

Comment:
I like this Objective section a lot!

Reply:
Wording it like this "library" "not a CLI" should definitely be carried into Hotel